ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -51,8 +51,14 @@ public SelenideElement getButton(String buttonTitle, SelenideElement differentRo
             fail(buttonTitle + " not found", e1);
         }
 
-        return differentRoot.shouldBe(visible).findAll(By.tagName("button"))
-                .filter(Condition.matchText("(\\s*)" + buttonTitle + "(\\s*)")).shouldHave(sizeGreaterThanOrEqual(1)).first();
+        ElementsCollection findedButtons = differentRoot.shouldBe(visible).findAll(By.tagName("button"))
+                .filter(Condition.matchText("(\\s*)" + buttonTitle + "(\\s*)")).shouldHave(sizeGreaterThanOrEqual(1));
+
+        if (findedButtons.size() > 1) {
+            fail("Ambiguous button title. Finded more that 1 button with title " + buttonTitle);
```

Comment:
`foundButtons`
`Found` 
:dancer:

Reply:
@avano I have already noticed this reprehensible typo mistake :cry: . I have already fixed it in the local repo.